sem_covid/entrypoints/etl_dags/finreg.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/python3
+
+# main.py
+# Date:  24/05/2021
+# Author: Laurentiu Mandru
+# Email: mclaurentiu79@gmail.com
+import json
+import logging
+from datetime import datetime, timedelta
+
+from SPARQLWrapper import SPARQLWrapper, JSON
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+
+from sem_covid import config
+from sem_covid.entrypoints.etl_dags.finreg_worker import DAG_NAME as SLAVE_DAG_NAME
+from sem_covid.services.store_registry import StoreRegistry
+
+logger = logging.getLogger(__name__)
+
+VERSION = '0.001'
+DATASET_NAME = "finreg_cellar"
+DAG_TYPE = "etl"
+DAG_NAME = DAG_TYPE + '_' + DATASET_NAME + '_' + VERSION
+CONTENT_PATH_KEY = 'content_path'
+CONTENT_KEY = 'content'
+FAILURE_KEY = 'failure_reason'
+RESOURCE_FILE_PREFIX = 'res/'
+TIKA_FILE_PREFIX = 'tika/'
+CONTENT_LANGUAGE = "language"
+FIELD_DATA_PREFIX = "works/"
+
+QUERY = '''PREFIX cdm: <http://publications.europa.eu/ontology/cdm#>
+PREFIX lang: <http://publications.europa.eu/resource/authority/language/>
+PREFIX res_type: <http://publications.europa.eu/resource/authority/resource-type/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX subj_mat: <http://publications.europa.eu/resource/authority/subject-matter/>
+PREFIX ev: <http://eurovoc.europa.eu/>
+PREFIX fd_030: <http://publications.europa.eu/resource/authority/fd_030/>
+
+SELECT DISTINCT
+    ?work
+    (group_concat(DISTINCT ?_title; separator="| ") as ?title)
```

Comment:
if we fetch the work id only, we can remove the tile and the whole block to expression&manifestation below.

Reply:
yes